assets/constants.ts

```diff
@@ -5361,6 +5361,16 @@ export = {
     "Omega": "Ω"
   },
 
+  "VALID_ALGEBRAIC_IDENTIFIERS": [
```

Comment:
This is still linked to `constants.GREEK_LETTER_NAMES_TO_SYMBOLS`, right? Maybe add a comment for that?

Reply:
Yup, done!